README.md

```diff
@@ -272,6 +275,17 @@ It may be useful to have some <tt>GRANT</tt> commands or similar run here when l
 table, the changes will be reverted and the backup table restored if post actions fail.</p>
     </td>
  </tr>
+ <tr>
+    <td><tt>avrocompression</tt></td>
+    <td>No</td>
+    <td>No compression (unless set in Hadoop config)</td>
+    <td>
+<p>Sets the compression codec to use on the Avro data to be loaded into Redshift. This overwrites the <tt>avro.output.codec</tt>
```

Comment:
Might be nice to enumerate the available options here, to save people researching how it's plumbed through. I believe `"snappy"`, `"bzip2`and `"gzip"` will all work.

Maybe it's worth us supporting an explicit `"none"` option for the sake of clarity? Now that I think about it, the empty string / null option to disable compression is a bit unnecessarily subtle?